Sources/Operation.swift

```diff
@@ -36,11 +49,17 @@ struct OperationBuilder: Builder {
     let responses: [Int : Reference<ResponseBuilder>]
     let defaultResponse: Reference<ResponseBuilder>?
     let deprecated: Bool
+    let operationId: String?
```

Comment:
For better index-ability you could add a doc-comment explaining what it is which also includes the OpenAPI name. E.g.
```
/// operationId - Unique string used to identify the operation. The id MUST be unique among all
/// operations described in the API. Tools and libraries MAY use the operationId to uniquely
/// identify an operation, therefore, it is recommended to follow common programming naming
/// conventions.
let identifier: String?
```